### DISCUSSION TITLE
Two different ArgoCD instances for managing the same set of clusters

### DISCUSSION THREAD

**Discussion by LiorLieberman** [argoproj/argo-cd#15396] [Q&A]
Has anyone ever tried or heard of a usecase where two ArgoCD instances (installed on different clusters) manage the same set of clusters ?
"manage" means they are able to deploy application to the same set of clusters.

Is this encouraged, or even possible?

**Comment by FernandoMiguel** [comment]
we have done it, during blue/green deployments of argo.
works fine, as long as the charts are the same.
one time there was a tiny issue in annotations between argo versions and that cause drift.
but other than that, argo is able to adopt existing resources based on annotations just fine

and if you dont want them to manage the same set of apps, even less problematic it is

**Comment by LiorLieberman** [comment]
Thanks for the input. 
I brought it in the contributors meeting as well - seems like this pattern, when used is mostly for migrations from one argo to another. (or blue/green of argo) as you said.

And of course, the key thing is that the charts must be the same. 

I am interested to learn if anyone had encountered a usecase where people use this specifically for a different purpose than migration.

**Reply by jkleinlercher** [reply]
@LiorLieberman did you then use this option in real use cases? I can think of a high-availability / multi-cloud topology where you want to use this pattern. However, I never tried this and so I don't have any experience with this.

**Reply by LiorLieberman** [reply]
I did not
